AIM/AIM/Domain/Entities/Track.m

```diff
@@ -0,0 +1,60 @@
+//
+//  Track.m
+//  AIM
+//
+//  Created by Oskar Szydlowski on 04.03.2018.
+//  Copyright © 2018 Oskar Szydlowski. All rights reserved.
+//
+
+#import "Track.h"
+
+@implementation Track
+
+-(id)initWithAttributesDictionary:(NSDictionary<NSString *,NSString *> *)attributes {
+    if (attributes[@"title"] == nil) {
```

Comment:
Same as above

Reply:
Also you should use `instancetype` instead of `id`.  cc @Vyeczorny to confirm

Reply:
I think in every place you use `id` you should use `instancetype`

Reply:
Actually, I thought it's exactly the opposite and should be (id) everywhere

Reply:
https://stackoverflow.com/a/14652187/1816253 check yellow box

Reply:
Afair it should be `instancetype`